app/v3/rand/internal/types/keys.go

```diff
@@ -0,0 +1,7 @@
+package types
+
+const (
+	ModuleName    = "rand"
```

Comment:
All `rand` should be renamed to `random` ?

Reply:
We'll postpone this refactoring to v1.0